app/src/main/java/net/squanchy/service/firebase/FirebaseDbService.kt

```diff
@@ -76,6 +76,7 @@ class FirebaseDbService(private val database: DatabaseReference) {
     private fun <T, V> observeChildAndEmit(path: String, clazz: Class<V>, map: (V?) -> T): Observable<T> {
         return Observable.create { emitter: ObservableEmitter<T> ->
             val listener = object : ValueEventListener {
+                @Suppress("TooGenericExceptionCaught") // We want to add info to *any* problems
```

Comment:
To be the best, we gotta pass the test
We gotta make it all the way
To the top of the mountain!
We can do it again!

Reply:
To feel the high
We got to learn to fly
We got to take it to the sky
On the wings of an eagle!

Reply:
Tho tbh I was thinking of:
> I wanna be the very best
> Like no one ever was
> To catch them is my real test
> To train them is my cause